### ISSUE TITLE
[Bug]: Version 0.5.25 synchronization via webdav Yandex disk does not work

### ISSUE THREAD

**Issue by MrSteelRat** [remotely-save/remotely-save#869]
### What happened?

Synchronization does not work when updating the plugin. The obsidian version was updated to the latest version, on version 0.23 Remote Save synchronization was successful on versions 0.25.5
![image](https://github.com/user-attachments/assets/beb5c287-99fc-43c2-b00c-007ce2fb2ecd)

### What OS are you using?

Windows, Linux, iOS (iPhone, iPad), Android

### What remote cloud services are you using? (Please choose the specified one if it's in the list)

webdav, others

### Version of the plugin

0.5.25

### Version of Obsidian

1.7.4

### Using password or not

- [X] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by fyears** [comment]
are you using remotely save pro for yandex, or using yandex webdav api?

**Comment by fyears** [comment]
and please export the sync plan to check what happened to the folder

**Comment by MrSteelRat** [comment]
i use yandex webadv, files not sync. Error sync it over and over

**Comment by zv09** [comment]
obviously, Yandex WebDAV having kind of issues on  0.5.25 with E2E encryption. 
I did rise another [bug report](https://github.com/remotely-save/remotely-save/issues/910) there with log... 
need an assistance from devs.. thanks
